src/coverlet.console/Logging/LogLevel.cs

```diff
@@ -0,0 +1,30 @@
+namespace Coverlet.Console.Logging
+{
+    /// <summary>
+    /// Defines logging severity levels.
+    /// </summary>
+    enum LogLevel
+    {
+        /// <summary>
+        /// Logs that track the general flow of the application. These logs should have long-term value.
+        /// </summary>
+        Detailed = 0,
+
+        /// <summary>
+        /// Logs that highlight an abnormal or unexpected event in the application flow, but do not otherwise cause the
+        /// application execution to stop.
+        /// </summary>
+        Normal = 1,
+
+        /// <summary>
+        /// Logs that highlight when the current flow of execution is stopped due to a failure. These should indicate a
+        /// failure in the current activity, not an application-wide failure.
+        /// </summary>
+        Minimal = 2,
+
+        /// <summary>
+        /// Not used for writing log messages. Specifies that a logging category should not write any messages.
+        /// </summary>
+        Quiet = 3
```

Comment:
`Quiet` should not hide warning and errors...I did some test with `dotnet test "D:\git\coverlet\test\coverlet.core.tests\coverlet.core.tests.csproj" -c Debug /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include=[coverlet.*]* -verbosity:quiet` and added some LogInfo/LogError and they show on output.

Reply:
done.